features/encrypt/core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptAlterTableTokenGenerator.java

```diff
@@ -134,26 +134,13 @@ private Collection<SQLToken> getModifyColumnTokens(final EncryptTable encryptTab
         for (ModifyColumnDefinitionSegment each : segments) {
             String columnName = each.getColumnDefinition().getColumnName().getIdentifier().getValue();
             if (encryptTable.isEncryptColumn(columnName)) {
-                result.addAll(getModifyColumnTokens(encryptTable.getEncryptColumn(columnName), each));
+                throw new UnsupportedOperationException("Unsupported operation 'modify' for the cipher column");
```

Comment:
Is it better to use ShardingSpherePreconditions?

Reply:
THX for point out.